apps/mobile/src/libs/components/button/button.tsx

```diff
@@ -1,19 +1,102 @@
 import React from "react";
-import { Pressable } from "react-native";
+import { Pressable, View } from "react-native";
+import LinearGradient from "react-native-linear-gradient";
```

Comment:
export from ~/libs/components/components
To avoid circular dependencies, create linear-gradient.ts in ~/libs/components and export from there as well